### ISSUE TITLE
Sample Import with Template - Coord error

### ISSUE THREAD

**Issue by ttir** [c3g/fms#57]
![Sample_Submission_ImportError02_Coord](https://user-images.githubusercontent.com/64210364/80110574-d376ac00-854c-11ea-8dbf-5a06411bd2e1.jpg)
[Sample_submission_Test_02_Coord.xlsx](https://github.com/c3g/fms/files/4523112/Sample_submission_Test_02_Coord.xlsx)

**Comment by dbujold** [comment]
The template still returns an error that "ID: This field cannot be blank.". Which ID column are we talking about?

**Comment by zxenia** [comment]
The template attached is an old one, there is no 'Individual Name' column anymore because we agreed to have 'Individual ID' instead.

**Comment by dbujold** [comment]
@zxenia Should the error message mention "Individual ID" instead of just ID?

**Comment by zxenia** [comment]
@dbujold That's true, it takes the database field name automatically and Individual ID is a template name. I think it will require to override the import-export source code. Let's leave the issue open.
